server/game/GameActions/StandCard.js

```diff
@@ -1,12 +1,17 @@
+const Message = require('../Message');
 const GameAction = require('./GameAction');
 
 class StandCard extends GameAction {
     constructor() {
         super('stand');
     }
 
+    message({ card }) {
+        return Message.fragment('stands {card}', { card });
+    }
+
     canChangeGameState({ card }) {
-        return ['faction', 'play area'].includes(card.location) && card.kneeled;
+        return card && ['faction', 'play area'].includes(card.location) && card.kneeled;
```

Comment:
Do you remember what problem required the addition of the `card &&` check? It seems like if there's a problem with null `card` objects being passed into game actions, it would apply to _all_ actions that take a card.

Reply:
I believe this was an old attempt to fix a concern of when `card` was null, but then found another way. Simply some code I forgot to revert, good catch!

Reverted in 31fa026dc6b1e7e530b9fea7fef084789d887b57

Can mark as resolved.